test/decorators/context.test.ts

```diff
@@ -0,0 +1,40 @@
+import { Context as AzureContext, HttpRequest } from '@azure/functions';
```

Comment:
Due to the decorator and Azure Context having the same name, this has to be done... ugly. Any ideas?

Reply:
No, this is completely fine IMO.